### DISCUSSION TITLE
"score" in the output file

### DISCUSSION THREAD

**Discussion by ojasvijain** [tbepler/topaz#195] [General]
Hi 

I just wanted to understand the "score" in the output file. From what I gather, it is the log likelihood of the particle extracted. Does this behave the same as confidence?

**Comment by tbepler** [comment]
Score is the log-likelihood ratio given by the classifier. This is equal to log(p(y=1)/p(y=0)). You can transform it into a probability with the sigmoid function. It is a confidence in the sense that it reflects the likelihood that this is a particle according to the model.
